GUI/src/components/NewServiceHeader/index.tsx

```diff
@@ -2,7 +2,7 @@ import { t } from "i18next";
 import React, { FC } from "react";
 import { Button, HeaderStepCounter, Track } from "..";
 import { EndpointData, PreDefinedEndpointEnvVariables } from "../../types/endpoint";
-import "../Header/Header.scss";
+import "@buerokratt-ria/header/src/header/Header.scss";
```

Comment:
**suggestion:** maybe we could think about not having styles and component in 2 different places and also create header as a module